spec/operators/map-spec.js

```diff
@@ -88,10 +116,12 @@ describe('Observable.prototype.map()', function () {
     });
 
     expectObservable(r).toBe(expected, values);
+    expectSubscriptions(a.subscriptions).toBe(asubs);
   });
 
   it('should map with index until completed', function () {
     var a = hot('-5-^-4--3---2----1--|--8--|--#');
```

Comment:
AFAIK RxJS 4 tests do this kind of thing too. I think it's testing whether the Subject behaves well even though we are pushing it beyond its limits, asking it to do what it cannot do.